peloton/peloton.py

```diff
@@ -377,7 +380,7 @@ def __init__(self, **kwargs):
         self.start_time = datetime.fromtimestamp(
             kwargs.get('start_time', 0), timezone.utc)
         self.end_time = datetime.fromtimestamp(
-            kwargs.get('end_time', 0), timezone.utc)
+            int(kwargs.get('end_time', 0) or 0), timezone.utc)
```

Comment:
Life's been hectic, so I haven't had a chance to look into the raw data coming back for this, but this seems reasonable at first pass to me. Only question I have is whether or not we need to look into adding similar logic (just in case) to `start_time`?

Reply:
It seems one can't get empty `start_time` - a ride simply won't exist without one. On the other hand, empty `end_time`, while an edge case, is very possible when the ride started but is still in progress.